pkg/policy/existing.go

```diff
@@ -206,12 +207,14 @@ func (pc *PolicyController) processExistingKinds(kind []string, policy *kyverno.
 		_, err := pc.rm.GetScope(k)
 		if err != nil {
 			gv, k := common.GetKindFromGVK(k)
-			resourceSchema, _, err := pc.client.DiscoveryClient.FindResource(gv, k)
-			if err != nil {
-				logger.Error(err, "failed to find resource", "kind", k)
-				continue
+			if !strings.Contains(k, "*") {
```

Comment:
`if k == *`?

Reply:
here we are checking against the complete kind,
Kind can be `g/*/k` or `*` so we cant use directly equal.